app/assets/stylesheets/_modules/_turbo-modal.scss

```diff
@@ -0,0 +1,108 @@
+@charset "UTF-8";
+@import "variables";
+
+/* ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯ *\
+  TURBO CONFIRM MODAL
+\* ˍˍˍˍˍˍˍˍˍˍˍˍˍˍˍˍˍˍ */
+dialog {
+  padding: 0;
+  border: none;
+}
+
+dialog:-internal-dialog-in-top-layer {
```

Comment:
d'ailleurs j'ai pas tiqué mais c'est quoi, `internal-dialog-in-top-layer` ? j'ai vu ça nulle part dans la spec, t'as trouvé ça où ?

Reply:
Dans l'inspecteur j'arrivais pas à bouger la modal dans le coin
<img width="287" alt="Screenshot 2024-05-12 at 19 32 32" src="https://github.com/vivesvoies/court-message/assets/16155041/47ba16d3-837d-49cd-8d09-1b02c1750ae7">

<img width="290" alt="Screenshot 2024-05-12 at 19 32 56" src="https://github.com/vivesvoies/court-message/assets/16155041/3ba2f5b9-61e4-476c-ad49-29891105503b">

Reply:
Je viens de test ca marche aussi bien sans ...

Reply:
oui, c'est une pseudo classe _interne_ à Chrome j'ai l'impression. MDN n'en parle pas.